bitnami/postgresql/README.md

```diff
@@ -230,7 +230,7 @@ $ kubectl delete pvc -l release=my-release
 | `readReplicas.priorityClassName`              | Priority Class to use for each pod (postgresql readReplicas)                                                                                              | `""`                        |
 | `readReplicas.extraInitContainers`            | Extra init containers to add to the pods (postgresql readReplicas)                                                                                        | `[]`                        |
 | `readReplicas.extraVolumeMounts`              | Extra volume mounts to add to the pods (postgresql readReplicas)                                                                                          | `[]`                        |
-| `readReplicas.extraVolumes`                   | Extra volumes to add to the pods (postgresql readReplicas)                                                                                                | `[]`                        |
+| `readReplicas.extraVolumes`                   | Extra volumes to add to the pods (postgresql readReplicas), the value is evaluated as a template.                                                                                               | `[]`                        |
```

Comment:
I think that you don't need to include this in the Readme. I took a look another charts like Redis and we are not including this information. So, please remove it. Thanks.